javascript/javascript1/week4/voice-assistant.js

```diff
@@ -0,0 +1,130 @@
+// VOICE ASSISTANT
+
+// Created an object, to keep, add and store all information about user in one place.
+// And for easy using a variables inside the function below.
+const speakerData = {
+  speakerName: "",
+  greeting: "Hello my name is Benjamin",
+  whatsName: "What is my name?",
+  addingFishingNote: "Add fishing to my todo",
+  addSinginInShower: "Add singing in the shower to my todo",
+  removingFishing: "Remove fishing from my todo",
+  whatIsInMyToDo: "What is on my todo?",
+  speakerToDoList: ["Make homework", "Make some food"],
+  speakerUpdatedToDoList: [],
+  dateToday: "What day is it today?",
+  simpleMath: ["-", "+", "*", "/"],
+};
+
```

Comment:
this const is ok, but only because it is homework.. and i am not sure it is doing you a service here either. 
fx. if i want to test your function by passing it 'Hello my name is Anne' it will return an error, but that is not because there's anything wrong with it. Rather it is because I need to update the variable above.. 
In short, be careful with such shortcuts!